src/model/ConnectionModel.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * ConnectionModel File Doc Comment
+ * 
+ * PHP Version 8.1.10
+ * 
+ * @category Model
+ * @package  App\model
+ * @author   Marine Sanson <marine_sanson@yahoo.fr>
+ * @license  https://opensource.org/licenses/gpl-license.php GNU Public License
+ * @link     https://www.blog.marinesanson.fr/ Not inline for the moment
+ */
+declare(strict_types=1);
+
+namespace App\model;
+
+/**
+ * ConnectionModel Class Doc Comment
+ * 
+ * @category Model
+ * @package  App\model
+ * @author   Marine Sanson <marine_sanson@yahoo.fr>
+ * @license  https://opensource.org/licenses/gpl-license.php GNU Public License
+ * @link     https://www.blog.marinesanson.fr/ Not inline for the moment
+ */
+class ConnectionModel
```

Comment:
```suggestion
class UserConnectionModel
```

Eventuellement tu peux enlever le suffixe Model puisque l'on sait que tu es dans le dossier model

Reply:
Corrigé. Je préfère garder Model quand même